robottelo/ui/syncplan.py

```diff
@@ -46,8 +46,8 @@ def create(self, name, description=None, startdate=None,
         if startdate:
             self.text_field_update(locators['sp.start_date'], startdate)
             self.wait_for_ajax()
-        self.submit_and_validate(
-            common_locators['create'], validation=submit_validate)
```

Comment:
please also remove 'submit_validate' from the function signature of this function as I see it is not used anymore.